Networking Sprint Challenge - PokeDex/Networking Sprint Challenge - PokeDex/Models/PokeMon.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  PokeMon.swift
+//  Networking Sprint Challenge - PokeDex
+//
+//  Created by Vijay Das on 2/1/19.
+//  Copyright © 2019 Vijay Das. All rights reserved.
+//
+
+
+// file for the PokeMon model
+
+import Foundation
+
+struct Pokemon: Codable {
+    
+    let name: String
+    let id: Int
+    let abilities: [Ability]
+    let type: [Type]
+    let sprites: SpriteImages
+
+ //   var imageData: Data?
```

Comment:
You can uncomment this as you will need an image to display the image of the pokemon.